samples/symbolication/README.md

```diff
@@ -0,0 +1,4 @@
+# Backtrace symbolization tool
+
+Produces more verbose backtrace symbolification with file name and line number.
+Only works on debug .dSYM binaries.
```

Comment:
Symbolication, symbolization or symbolification? There are all three of them in the changes.